docker-compose.yml

```
@@ -1,62 +1,37 @@
 version: '3.9'
 services:
   cache:
-    container_name: backend_cache
-    image: redis:6.2.5-alpine
-    command: redis-server --appendonly yes
-    volumes:
-      - cache_data:/var/lib/redis/data
+    image: redis:alpine
+    restart: on-failure
     ports:
-      - '6379:6379'
+      - '6380:6379'
```

why remap the port to 6380?

```suggestion
      - '6379:6379'
```

---

Yeh this is just force of habit.. I like to keep the default service ports open on my machine. It's pretty rare that I have any one service that can listen on the default port anyway as if I have our full stack running I've usually got a few databases, and lots of services up.. so the first thing I do when I start a new project is assign the project an unused port that is an increment on the last one. Again, I'll change all those back after I fork off.